torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -2179,6 +2179,15 @@ at::Tensor& XLANativeFunctions::mv_out(const at::Tensor& self,
   return out;
 }
 
+at::Tensor XLANativeFunctions::nan_to_num(const at::Tensor& self,
+                                          c10::optional<double> nan,
```

Comment:
Let's resolve the optional here instead of passing to XLATensor, you can use `XlaHelpers::MinMaxValues` to get the min max for a certain type.

Reply:
Changed.